lib/kernel/src/trace.erl

```diff
@@ -812,7 +809,7 @@ Argument **`FlagList`** is a list of options. The following are the valid option
 
 - **`global`** - Turn on or off call tracing for global function calls (that
   is, calls specifying the module explicitly). Only exported functions match and
-  only global calls generate trace messages. **This is the default**.
```

Comment:
Why remove that `global` is the default?

Reply:
No good reason/force of habit. Sorry. I've been deleting similar phrases from other parts of the documentation where it no longed made sense because description that used be common for several clauses now only describes a single clause.

I failed to realize that this applies to `ArgList` being empty.

I will fix.